.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -0,0 +1,14 @@
+<component name="InspectionProjectProfileManager">
```

Comment:
I'm pretty sure that this file should not be part of the code base. Please remove

Reply:
Yup, Done!